website/docs/docs/workshops/ci-cd-legacy/set-up-ci/04_configure_webhook.md

```diff
@@ -0,0 +1,92 @@
+---
+id: configure-webhook
+title: Configure webhook
+---
+
+When using Pact in a CI/CD pipeline, there are two reasons for a pact verification task to take place:
+
+* When the provider changes (to make sure it does not break any existing consumer expectations)
+* When a pact changes (to see if the provider is compatible with the new expectations)
+
+To ensure that the verification step is run whenever a pact changes, we need to configure a Pactflow webhook to trigger a provider verification build in Github Actions. You can see the configuration for this build in `.github/workflows/verify_changed_pact.yml` in the provider project. Rather than verifying the pacts with the configured tags, this build just verifies the pact that has changed. This is achieved by passing the URL of the changed pact to the build via a parameter in the body of the webhook request.
+
+The Pactflow webhook will need a Github access token to be able to trigger the build in Github. We don't want the Github token to be stored in clear text in the webhook, so we will create a secret in Pactflow to contain token.
+
+1. Create a Github token.
+    1. In Github:
+        1. Open the `Personal acesss tokens page`
```

Comment:
@YOU54F while you are making changes, do you want to fix this typo as well? acesss->access? its in both legacy and new